resources/arduino_files/libraries/dallas-temperature-control/README.panoptes

```diff
@@ -0,0 +1,4 @@
+This is commit 9cc81f0599f5832f10560939ddea4a70f44b9893
```

Comment:
Should we make a README up at the level of the arduino_files/ folder that describes each one so people don't have to go digging?

Reply:
I'm inclined to say put one in each of the libraries sub-folders, so I remember to update it when I am updating a library. I don't think it is too much digging. OK?